src/EdFi.AnalyticsMiddleTier.DataStandard33/EPP/MSSQL/0002-View-FinancialAid-Create.sql

```diff
@@ -0,0 +1,35 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+IF OBJECT_ID('[analytics].[EPP_FinancialAidFact]') IS NOT NULL
+	DROP VIEW [analytics].[EPP_FinancialAidFact]
+
+GO
+
+CREATE VIEW [analytics].[EPP_FinancialAidFact] AS
+
+---Financial Aid
+SELECT   tpdm.Candidate.CandidateIdentifier

```

Comment:
In AMT in general, for every view, we define a unique identifier field which we suffix with a `key`. From what I have seen, in most cases for this new collection the `CandidateIdentifier` works as a unique identifier. Maybe it's your case as well, and maybe not. Take for example `analytics.SectionDim` view and `analytics.ClassPeriodDim` view. `analytics.SectionDim` has a `SectionKey` field, and `analytics.ClassPeriodDim` has a `ClassPeriodKey` field. It's basically a concatenation of key columns that will produce a unique identifier for the results.